spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/TraceRestTemplateInterceptorTests.java

```diff
@@ -29,7 +29,7 @@
 import brave.propagation.ThreadLocalCurrentTraceContext;
 import brave.sampler.Sampler;
 import brave.spring.web.TracingClientHttpRequestInterceptor;
-import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Why change from `lang` to `lang3`?

Reply:
I think it was a transitive dependency. I don't think we should be using commons lang at all.